### PR TITLE
PR: Cart

### PR DESCRIPTION
**Pull request by pavan6115**
- Implemented Cart Functionalities
  -  Add to Cart 
  - Remove From Cart
  - Cart Item Hander [Increment/Decrement]
  - Notification Count in Header 
  
 

https://user-images.githubusercontent.com/55844228/187013207-3d7ccdf3-48c0-4d10-a36b-e165e8c857ce.mov

### CODE REVIEW

**Review by jatinmore — APPROVED** [review]
looks good to me

File: src/reducer/cart/cart-reducer.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                ...state,
+                cart: state.cart.map((item) => {
+                    if (action.payload._id === item._id) {
+                        return {
+                            ...item, 
+                            qty: item.qty + 1
+                        }
+                    }
+                    return item
+                }),
```
Comment (viru2001) [inline src/reducer/cart/cart-reducer.js:29]: you can use ternary operator here

**Review by viru2001 — APPROVED** [review]
looks good to merge

File: src/reducer/cart/cart-reducer.js (hunk 1 above)
Comment (pavan6115) [reply src/reducer/cart/cart-reducer.js:29]: ok 👍

**Pull request merged**